apps/api/src/secret/service/secret.service.ts

```diff
@@ -505,16 +505,11 @@ export class SecretService {
 
     for (const secret of secrets) {
       // Decrypt the secret value
-      for (let i = 0; i < secret.versions.length; i++) {
-        const version = secret.versions[i]
-        // Optionally decrypt secret value if decryptValue is true
-        if (decryptValue) {
-          const decryptedValue = await decrypt(
-            project.privateKey,
-            version.value
-          )
-          version.value = decryptedValue
-        }
+      const version = secret.versions[0]
```

Comment:
Make the following changes:

- Rename `version` to `latestSecretVersion`
- Move the line inside the `if` block